src/isar/apis/models/start_mission_definition.py

```diff
@@ -33,12 +35,14 @@ class StartMissionInspectionDefinition(BaseModel):
 
 
 class StartMissionTaskDefinition(BaseModel):
+    id: Optional[str]
     pose: InputPose
     tag: Optional[str]
     inspections: List[StartMissionInspectionDefinition]
 
 
 class StartMissionDefinition(BaseModel):
+    id: Optional[str]
```

Comment:
I think `Optional` parameters should be after all the mandatory parameters

Reply:
fixed